packages/dev/docs/pages/releases/2021-11-24.mdx

```diff
@@ -0,0 +1,160 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: As we head into the holiday season, our latest release focuses on kick-starting some of our pre-release components as well a whole list of fixes, and to top it all off, a new selection mode! 
+date: 2021-11-24
+---
+
+# November 24, 2021 Release
+
+As we head into the holiday season, our latest release focuses on kick-starting some of our pre-release components as well a whole list of fixes, and to top it all off, a new selection mode! 
+
+Our `@react-aria/interactions` package now includes a new `useLongPress` hook for handling long press interactions across both mouse and touch devices. This hook automatically supports preventing text selection and context menus, allowing customizable time thresholds, and an accessibility description to help discoverability with assistive technology users. Take a look at the documentation in our [React Aria docs](https://react-spectrum.adobe.com/react-aria/useLongPress.html). 
```

Comment:
Link `useLongPress` text to docs?

```suggestion
Our `@react-aria/interactions` package now includes a new [useLongPress](https://react-spectrum.adobe.com/react-aria/useLongPress.html) hook for handling long press interactions across both mouse and touch devices. This hook automatically supports preventing text selection and context menus, allowing customizable time thresholds, and an accessibility description to help discoverability with assistive technology users. Take a look at the documentation in our [React Aria docs](https://react-spectrum.adobe.com/react-aria/useLongPress.html). 
```

Reply:
Oh I see it's later. I was confused. I guess I assumed "React Aria docs" just linked to the react-aria homepage...